core/build.gradle

```diff
@@ -56,4 +56,4 @@ dependencies {
     implementation 'io.reactivex:rxjava:1.1.4'
 
     testImplementation 'junit:junit:4.12'
-}
+}
```

Comment:
Remove this change.

Reply:
I can't understand what is the change here. The closing braces are at the same position and line number as earlier.

Reply:
@luckyman20 i unstaged a particular file using ` git checkout HEAD^ path/of/file `

Reply:
But still after that these changes seem to remain.